metadata-ingestion/src/datahub/ingestion/source/aws/glue.py

```diff
@@ -71,6 +75,29 @@ def check_direction(cls, v: str) -> str:
             )
         return v.lower()
 
+    @validator("underlying_platform")
+    def underlying_platform_validator(cls, v: str) -> str:
+        if not v or v in VALID_PLATFORMS:
+            return v
+        else:
+            raise ConfigurationError(
+                f"'underlying_platform' can only take following values: {VALID_PLATFORMS}"
+            )
+
+    @validator("platform")
+    def platform_validator(cls, v: str) -> str:
+        if not v or v in VALID_PLATFORMS:
+            return v
+        else:
+            raise ConfigurationError(
+                f"'platform' can only take following values: {VALID_PLATFORMS}"
+            )
+
+    @validator("platform_instance")
+    def platform_instance_validator(cls, v):
```

Comment:
Delete this validator. This connector should be able to rely on any validation added for `platform_instance` in the future by `PlatformSourceConfigBase`.